src/DynamoCore/Migration/Migration.cs

```diff
@@ -1360,6 +1354,7 @@ public class NodeMigrationAttribute : Attribute
         /// <summary>
         /// Version this migrates to.
         /// </summary>
+        [Obsolete("All migrations are now processed, beginning with the first migration whose version is >= the workspace version. The To property will be ignored.", false)]
```

Comment:
That's open for discussion. I can just rip it out, and take out all references to `to:` as well. I started to do that yesterday and thought better of it. But we _are_ making a major version change.